source/server/server.cc

```diff
@@ -304,6 +304,9 @@ void InstanceImpl::updateServerStats() {
 
 void InstanceImpl::flushStatsInternal() {
   updateServerStats();
+  // Update certificate stats for each context.
+  ssl_context_manager_->iterateContexts(static_cast<std::function<void(Envoy::Ssl::Context&)>>(
```

Comment:
Move this to `updateServerStats()`; I think that will be a little bit cleaner

Reply:
Good idea. Moved